README.md

```diff
@@ -142,6 +140,38 @@ The figure below shows the difference between interpolating the corrections or n
 
 ![Corrected High Rate Poses With Interpolation](docs/CorrectedHighRatePosesWithInterpolation.png)
 
+### poses_to_poses_file
+
+This tool can be used to convert a pose file to another pose file of any type (txt, json, ply, pcd). Additionally, there is the option to specify the frame in which poses are to be expressed. This is useful when comparing SLAM trajectories, as large offsets between differently assumed base_link frames can lead to incorrect comparisons.
+
+For more information on how to run the executable, run:
+
+```
+./path_to_build_dir/3d_map_builder_poses_to_poses_file --help
+```
+
+### manual_calibration
+
+This tool can be used to manually adjust the extrinsic calibration of a candidate sensor (ex. multibeam sonar, vertically oriented lidar) with respect to a reference sensor (ex. horizontally oriented lidar). To begin, launch:
+
+```
+roslaunch map_builder manual_calibration.launch
+```
+
+and run the executable. For more information on how to run the executable, run:
+
+```
+./path_to_build_dir/manual_calibration.cpp --help
+```
+
+This will bring up a custom rviz window, where `/manual_calibration/reference_cloud` and `/manual_calibration/candidate_cloud` topics are subscribed to and visualized. The `reference_cloud` and `candidate_cloud` topics have constructions of the user specified reference sensor and candiate sensor, respectively, published over a window of size `window_size` from a starting index `start_index`. These manually adjustable parameters are available in the rqt_ez_publisher window that opens upon launch, with  `x,y,z,roll,pitch,yaw` parameters available for manual extrinsic calibration. When checking off `save_data`, reference and candidate clouds are published as shown below.
+
```

Comment:
save data just saves or publishes the cloud results? How do you save the calibration?

Reply:
I will change `save_data` to `publish` as this option simply publishes the clouds when any of the the parameters (i.e. `start_index`, `window_size`, `x,y,z,roll,pitch,yaw`) changes (see Algorithm 3 in the IEEE submission for specifics). Right now, there is no way of actually saving the calibration since this was sufficient for the paper. Ideally, there would be functionality to write calibrations to a file, though that would take some developement which I unfortunately cannot spend time on right now. The hand calculations take only a few minutes to do, though for someone not familiar with the code structure it would be a good feature.